lib/boom_notifier/error_info.ex

```diff
@@ -80,4 +79,35 @@ defmodule ErrorInfo do
     |> Tuple.to_list()
     |> Enum.join(".")
   end
+
+  @spec build_custom_data(map(), custom_data_strategy_type) :: map()
+  defp build_custom_data(_conn, :nothing), do: %{}
```

Comment:
Returning %{} here causes that the `Data:` line in the email templates is always displayed. Maybe we should return nil here?